docs/preset-env.md

```diff
@@ -264,18 +264,85 @@ This option is useful for "blacklisting" a transform like `@babel/plugin-transfo
 
 `"usage"` | `"entry"` | `false`, defaults to `false`.
 
-This option configures how `@babel/preset-env` handles polyfills.
+This option configures how `@babel/preset-env` handles polyfills and whether optimizations are applied to only include polyfills that are needed.
```

Comment:
What about something like:

```suggestion
This option configures `@babel/preset-env` to only include the polyfills needed by your target environment(s). This can either include all polyfills needed to mimic a ES2020 environment (`entry`), or only include polyfills as they are needed/used in each file (`usage`).
```

Reply:
Riffing on that a bit more, how about this? I dropped mention of ES2020 because that will become outdated eventually.

```suggestion
This option configures `@babel/preset-env` to only include the polyfills needed by your target environment(s) to mimic the latest JS-spec environment. Options are to include polyfills as they are needed/used in each file (`usage`) or include polyfills at the entrypoint to your application (`entry`).
```

Given the nuance you mention, I'm not exactly clear on what are the pros/cons of each? If `usage` wouldn't help people in the case about `node_modules` you mentioned above, is there a reason people wouldn't always just use `entry`? If I understand that aspect more, then I think we could get a better synopsis out to users.

Reply:
There are certainly people who _do_ compile their `node_modules` (or if you're sure that your deps don't need to be polyfilled) to get the full benefit of `usage`, it's just a caveat that people may not be aware of when they're setting up Babel for their project.

Reply:
That totally makes sense. I'm just trying to understand, when you say "the full benefit of `usage`", what is that benefit? Are there benefits of `usage` over `entry`? I think that would be good to showcase.